src/test/java/uk/gov/register/functional/ApplicationTest.java

```diff
@@ -20,7 +20,7 @@
 
 public class ApplicationTest {
     @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    public static RegisterRule register = new RegisterRule();
```

Comment:
Would it be possible to pass in `registerName` as 1 of the constructors to then skip passing register name in `getRequest` or `targetRegister` overloads (just a sugah really).